src/OpenLoco/Windows/CompanyWindow.cpp

```diff
@@ -297,7 +297,8 @@ namespace OpenLoco::Ui::Windows::CompanyWindow
                 case widx::change_owner_name:
                 {
                     auto company = CompanyManager::get(CompanyId(self.number));
-                    TextInput::openTextInput(&self, StringIds::title_name_owner, StringIds::prompt_enter_new_name_for_owner, company->ownerName, widgetIndex, nullptr);
+                    auto inputSize = StringManager::kUserStringSize - 1;
+                    TextInput::openTextInput(&self, StringIds::title_name_owner, StringIds::prompt_enter_new_name_for_owner, company->ownerName, inputSize, widgetIndex, nullptr);
```

Comment:
One final comment (hopefully). We have made `StringManager::kUserStringSize - 1` the default for `InputSession`. I think would make sense to instead make it the default for `TextInput::openTextInput`. This also means moving the parameter to the last position, but that should be an easy operation in most IDEs.

This `inputSize` parameter can then be omitted everywhere where `StringManager::kUserStringSize - 1` is intended to be the limit. In other words, only in three places will you actually need to pass the parameter.

Sorry, I don't mean to